### PR TITLE
output installed version number after setup

### PR DESCRIPTION
**Pull request by robindaumann**
implements #50 

Example run with changes: [Actions](https://github.com/robindaumann/potential-barnacle/commit/cdc009b48e76c64438854612a36e8c07a46ac041/checks?check_suite_id=390026563)

### CODE REVIEW

**Comment by sburns** [comment]
I'd like to see this go a tad further and write the version as an [action output](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#outputs). This way, we _should_ be able to reference that output in later steps. Specifically, I'd like to be able to build a virtualenv cache key based on the installed version of python.

**Comment by robindaumann** [comment]
What format should the output have? Like in the install dir e.g. `Python/3.8.0` or like in the console output `Python 3.8.0`?

**Comment by sburns** [comment]
Personally I'd prefer no slashes or spaces. `3.7.6`, `3.6.9`, `3.8.0`, `pypy2`, etc. The goal is to relay the specific version installed for cases that we ask for a non-specific version, like `3.7.X`.

**Comment by robindaumann** [comment]
That was exactly my use case. I will implement the output in this pr.

File: src/setup-python.ts [hunk 1]
```diff
@@ -7,7 +7,10 @@ async function run() {
     let version = core.getInput('python-version');
     if (version) {
       const arch: string = core.getInput('architecture', {required: true});
-      await finder.findPythonVersion(version, arch);
+      const installed = await finder.findPythonVersion(version, arch);
+      console.log(
+        `Successfully setup ${installed.impl} (${installed.version}).`
```
Comment (konradpabjan) [inline src/setup-python.ts]: The extra `.` seems unnecessary 

```suggestion
        `Successfully setup ${installed.impl} (${installed.version})`
```

**Comment by konradpabjan** [comment]
Tested out your changes, looks good 👍

Better output/logging is always good and this is a pretty small addition. Example output can be seen here: https://github.com/konradpabjan/Testing2/runs/489889083?check_suite_focus=true

@brcrista @madhurig any thoughts or objections?

**Comment by konradpabjan** [comment]
Thanks for the contribution @robindaumann 

I'll go ahead and merge your changes in. The small update that I suggested I'll address myself before pushing out a new release

**Comment by robindaumann** [comment]
Alright thanks :)

**Pull request merged**